feedr-starter-code/js/app.js

```diff
@@ -1,3 +1,247 @@
 /*
   Please add all Javascript code to this file.
-*/
\ No newline at end of file
+*/var Newsurl = 'https://newsapi.org/v2/top-headlines?country=us&apiKey=628fd7b3d44b4d67a682072cba2d449d'
+  var GardianURL  = "https://content.guardianapis.com/search?q=debate&tag=politics/politics&from-date=2014-01-01&api-key=a00373e0-59ca-49e0-ae02-cb3efe9c189e";
+  var digg = 'https://newsapi.org/v2/top-headlines?sources=google-news&apiKey=628fd7b3d44b4d67a682072cba2d449d'
+
+  var appid = "628fd7b3d44b4d67a682072cba2d449d";
```

Comment:
It's better if you put the APIs keys in an external file and not push it to Github!